tst/EnergyPlus/unit/ChillerExhaustAbsorption.unit.cc

```diff
@@ -637,29 +638,60 @@ TEST_F(EnergyPlusFixture, ExhAbsorption_calcHeater_Fix_Test)
-    state->dataLoopNodes->Node(3).MassFlowRate = 0.5;
-    state->dataLoopNodes->Node(4).TempSetPoint = 70.0;
-    state->dataLoopNodes->Node(7).Temp = 350.0;
-    state->dataLoopNodes->Node(7).MassFlowRate = 0.5;
+    auto &hwPlantLoop = state->dataPlnt->PlantLoop(1);
+    hwPlantLoop.FluidName = "WATER";
+    hwPlantLoop.FluidIndex = 1;
+    hwPlantLoop.LoopDemandCalcScheme = DataPlant::LoopDemandCalcScheme::SingleSetPoint;
+    hwPlantLoop.LoopSide(DataPlant::LoopSideLocation::Demand).FlowLock = DataPlant::FlowLock::Locked;
+
+    EXPECT_EQ(1, thisChillerHeater.ChillReturnNodeNum);
+    EXPECT_EQ("EXH CHILLER INLET NODE", state->dataLoopNodes->NodeID(1));
+    EXPECT_EQ(2, thisChillerHeater.ChillSupplyNodeNum);
+    EXPECT_EQ("EXH CHILLER OUTLET NODE", state->dataLoopNodes->NodeID(2));
+    EXPECT_EQ(3, thisChillerHeater.HeatReturnNodeNum);
+    EXPECT_EQ("EXH CHILLER HEATING INLET NODE", state->dataLoopNodes->NodeID(3));
+    EXPECT_EQ(4, thisChillerHeater.HeatSupplyNodeNum);
+    EXPECT_EQ("EXH CHILLER HEATING OUTLET NODE", state->dataLoopNodes->NodeID(4));
+    EXPECT_EQ(5, thisChillerHeater.CondReturnNodeNum);
+    EXPECT_EQ("EXH CHILLER CONDENSER INLET NODE", state->dataLoopNodes->NodeID(5));
+    EXPECT_EQ("CAPSTONE C65 COMBUSTION AIR INLET NODE", state->dataLoopNodes->NodeID(6));
+    EXPECT_EQ(7, thisChillerHeater.ExhaustAirInletNodeNum);
+    EXPECT_EQ("CAPSTONE C65 COMBUSTION AIR OUTLET NODE", state->dataLoopNodes->NodeID(7));
+
+    constexpr Real64 hwSupplySetpoint = 70.0;
+    constexpr Real64 hwReturnTemp = 60.0;
+    constexpr Real64 hwMassFlow = 0.5;
+    state->dataLoopNodes->Node(thisChillerHeater.HeatReturnNodeNum).Temp = hwReturnTemp;
+    state->dataLoopNodes->Node(thisChillerHeater.HeatReturnNodeNum).MassFlowRate = hwMassFlow;
+    state->dataLoopNodes->Node(thisChillerHeater.HeatSupplyNodeNum).TempSetPoint = hwSupplySetpoint;
+
+    // Minimum temperature leaving the Chiller absorber is 176.6 C (350 F)
+    constexpr Real64 exhaustInTemp = 350.0;
+    constexpr Real64 absLeavingTemp = 176.667;
+    constexpr Real64 exhaustInMassFlowRate = 0.5;
+    constexpr Real64 exhaustInHumRate = 0.005;
+    state->dataLoopNodes->Node(thisChillerHeater.ExhaustAirInletNodeNum).Temp = exhaustInTemp;
+    state->dataLoopNodes->Node(thisChillerHeater.ExhaustAirInletNodeNum).MassFlowRate = exhaustInMassFlowRate;
+    state->dataLoopNodes->Node(thisChillerHeater.ExhaustAirInletNodeNum).HumRat = exhaustInHumRate;
 
+    Real64 loadinput = 5000.0;
+    bool const runflaginput = true;
     thisChillerHeater.calcHeater(*state, loadinput, runflaginput);
 
-    EXPECT_NEAR(thisChillerHeater.HeatingLoad, 21085.0, 1e-6);
+    const Real64 CpHW = FluidProperties::GetSpecificHeatGlycol(*state, hwPlantLoop.FluidName, hwReturnTemp, hwPlantLoop.FluidIndex, "UnitTest");
+    EXPECT_EQ(4185.0, CpHW);
+    const Real64 expectedHeatingLoad = (hwSupplySetpoint - hwReturnTemp) * hwMassFlow * CpHW;
+
+    EXPECT_NEAR(20925.0, expectedHeatingLoad, 1e-6);
```

Comment:
Recalculate the expected heating load properly, with a CpHW that is properly calculated at 60°C and not at 0°C always.